src/components/dashboard/Dashboard.js

```diff
@@ -367,6 +370,8 @@ const Dashboard = props => {
         }),
       ]).start()
     }
+
+    setTimeout(() => setGetNextFeedAllowed(true), 300)
```

Comment:
this is called throttling, use lodash throttle with useCallback instead of the state variable

Reply:
@sirpy 

I already tried to use the throttle and also debounce functions as you said but the best behavior is reached by the implementation which I pushed. 
When using throttle or debounce there's still some extra getNextFeed fn executions.

Reply:
this sounds impossible, unless you didnt use useCallback to memoize the throttled method.
what you are looking for is probably throttle, it can limit the call to once every 300ms
debounce waits X time until no more calls to the method and then executes the last call

Reply:
the same think - you're using timeout instead to add some callback that executes when animation actually ends